website/case-studies/user-insights-on-spatially-explicit-forest-carbon-budget-model-workflow.md

```diff
@@ -0,0 +1,287 @@
+
+![In other cases, the year of stand establishment matches a disturbance event. We know when the stand was established, but we do not know the age of the stand before the disturbance event. Determining the pre-disturbance stand age can be accomplished using an age probability distribution or through various other techniques](../static/img/disturbance-event.png)
+
+> In other cases, the year of stand establishment matches a disturbance event. We know when the stand was established, but we do not know the age of the stand before the disturbance event. Determining the pre-disturbance stand age can be accomplished using an age probability distribution or through various other techniques.
+
+![ If we are trusting the inventory information, it may happen that we need to create a disturbance because no event is present.](../static/img/no-disturbance-event.png)
+
+>  If we are trusting the inventory information, it may happen that we need to create a disturbance because no event is present.
+
+There are a wide variety of additional rollback situations that can occur. The purpose of mentioning it here is to highlight that a process is required to harmonize inventory age information and disturbance event information, so that there is a sensible sequence of events and parameters passed to GCBM.
+
+The process typically involves stacking up various input data such as inventories of various vintages, different disturbance events, and then decided on a set of rules to resolve conflicts. When data is large and complex, the stacking and conflict resolution is easiest to handle when the inputs are simple raster arrays. In array format they can be handled with database tools and powerful Python numpy libraries.
+
+For simple and small projects, rollback procedures are feasible within a GIS package and with basic GDAL tools.
+
+The end goal of a rollback process is the following:
+
+1. An age layer at the start of the simulation that makes sense.
+2. Disturbance layers that have been adjusted to match the inventory, OR, an inventory that has been adjusted to match the disturbance information.
+3. If required, transition rules to help resolve disturbance/inventory discrepancies.
+
+### Future Events
+
+Since GCBM/FLINT requires spatially described events, or map data, future event projections must be projected on to the forest landscape. Various Python/GDAL/ArcGIS tools can be used to accomplish this, and the procedures can range from placing events with a simple “birdshot” style blasts, to careful placement based on analysis of various spatial supporting layers. To-date, CFS has developed various versions of tools that project future harvest and future wild fire events. Some consist of open-source Python libraries and GDAL tools, while others make use of ArcGIS tools or paid optimization packages. Generic future projection tools have not yet been developed.
```

Comment:
I find the 'birdshot' analogy here kind of confusing. I think they mean that prior knowledge about the spatial distribution of events can be uniform, consist of a random sample or be determined from ancillary data.